### ISSUE TITLE
Post title in inbox

### ISSUE THREAD

**Issue by Mizu-6079** [liftoff-app/liftoff#191]
I wish we were able to see which post someone was commenting on in the inbox

**Comment by varaki** [comment]
Yes, it would be nice. Maybe taking it further and clicking on it would take you to the post and comment too, with that comment highlighted, so you would get a context as well.
I think infinity had this feature, but currently I think the official lemmy-ui doesn't have this, so it may be worth to request it there (as well)

**Comment by Mizu-6079** [comment]
> I think infinity had this feature, but currently I think even the official lemmy-ui doesn't have this, so it may be worth to request it there (as well)

Jerboa a feature where you can see the post the reply is on. But if they're replying to a comment you can't see which comment it is.
